aws_account_policies/aws_password_policy_complexity_guidelines.py

```diff
@@ -2,23 +2,22 @@
 
 
 def policy(resource):
-    if not resource[
-            'RequireUppercaseCharacters'] and 'RequireUppercaseCharacters' not in IGNORED:
+    if not resource["RequireUppercaseCharacters"] and "RequireUppercaseCharacters" not in IGNORED:
```

Comment:
Did we also want to do `.get()` access in policies?

Reply:
Top level keys in resources are always present, no exceptions. For that reason I'm kind of ok with using square brackets in policies. However, we should 100% use `deep_get` for any nested lookups in policies.